library/src/meta/bibliography.rs

```diff
@@ -135,7 +135,7 @@ impl Show for BibliographyElem {
             seq.push(
                 HeadingElem::new(title)
                     .with_level(NonZeroUsize::ONE)
-                    .with_numbering(None)
+                    .with_numbering(HeadingElem::numbering_in(styles))
```

Comment:
if we just want to respect the active style, completely removing the `with_numbering` call does that. I think it's not that common to number the bibliography, but I guess we change make that change since it's simple to disable it with `show bibliography: set heading(numbering: none)`

Reply:
on a further note: the same thing is in `meta/outline.rs`. do we also want to change it there?

Reply:
I think so. I also changed it there.